terraform/hcl/applicable/req_sw_terraform_aws_athena_encrypt_workgroup.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_athena_workgroup" "vulnerable_example" {
+    name = "dummy"
+    # encryption_configuration is not set
+}
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 4 2 vulnerable_example /terraform/hcl/applicable/req_sw_terraform_aws_athena_encrypt_workgroup.tf)

### 🛠️ Infrastructure as Code Vulnerability
	
Severity: ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableMediumSeverity.png)<br>  medium

Finding: `AWS Athena workgroup does not usedata-at-rest encryption`

#### 👇 Details

<details>
<summary> <b>Full description</b> </summary>
<br>
Resource `aws_athena_workgroup` should have key `encryption_configuration`

Vulnerable example - 
```
resource "aws_athena_workgroup" "vulnerable_example" {
      # encryption_configuration is not set
}
```

Secure example -
```
resource "aws_athena_workgroup" "secure_example" {
  configuration {
    result_configuration {
      encryption_configuration {
        encryption_option = "SSE_KMS"
        kms_key_arn       = aws_kms_key.example.arn
      }
    }
  }
}
```

</details>

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)